test/runtests.jl

```diff
@@ -477,5 +478,22 @@ using Compat
             @test_broken isready(bottom_dc3) == true
             @test_broken fetch(bottom_dc3) == 14
         end
+
+        @testset "DeferredFuture parallel serialization" begin
+            pnums = addprocs(2)
+            @everywhere using Dispatcher
+
+            try
+                ctx = DispatchContext()
+                exec = ParallelExecutor()
+
+                op = Op(()->3)
+                add!(ctx, op)
+
+                results = run!(exec, ctx)
```

Comment:
Hmmm, yeah, I'm not entirely sure either. Did a bit of digging though and you could probably adapt the base julia tests for serialization cycles https://github.com/JuliaLang/julia/blob/master/test/serialize.jl#L350 ?